fiftyone/core/dataset.py

```diff
@@ -54,6 +54,13 @@
 logger = logging.getLogger(__name__)
 
 
+class MissingDatasetError(ValueError):
```

Comment:
nit: suggest `DatasetNotFoundError` for consistency with how the error message is worded

"Missing" implies that I need to go looking for it 😉

Reply:
Good idea!